packages/buttons/src/styled/StyledIconButton.ts

```diff
@@ -38,6 +38,7 @@ const iconButtonStyles = (props: IStyledButtonProps & ThemeProps<DefaultTheme>)
     border: ${props.isBasic && 'none'};
     padding: 0;
     width: ${getHeight(props)};
+    min-width: ${getHeight(props)};
```

Comment:
[nit] it might be slightly more efficient/maintainable to pull the two `getHeight` function calls into a `const width = getHeight(props);` variable substitution declared above the `css` block.